semantic.cabal

```diff
@@ -313,13 +312,11 @@ library
                      , unordered-containers ^>= 0.2.9.0
                      , vector ^>= 0.12.0.2
                      , tree-sitter-go ^>= 0.2
-                     , tree-sitter-json ^>= 0.2
                      , tree-sitter-php ^>= 0.2
-                     , tree-sitter-python ^>= 0.5
+                     , tree-sitter-python ^>= 0.6
```

Comment:
We still have to have a direct dependency on `tree-sitter-python` for the assignment, which uses the `Grammar` datatype.